code/modules/admin/verbs/greytide_worldwide.dm

```diff
@@ -0,0 +1,138 @@
+/client/proc/greytide_worldwide(var/mob/user)
+	if(!ticker)
+		alert("The game hasn't started yet!")
+		return
+
+	var/list/pickfrom = list("Cancel")
+	for(var/datum_type in typesof(/datum/greytide_worldwide))
+		var/datum/greytide_worldwide/D = datum_type
+		var/name = initial(D.name)
+		if(name)
+			pickfrom += name
+			pickfrom[name] = D
+
+	var/event_type_name = input(user, "Select an event.", "GREYTIDE WORLDWIDE", null) in pickfrom as text|null
+	if(!event_type_name)
+		return
+	var/event_type = pickfrom[event_type_name]
+	if(!event_type)
+		return
+
+	var/datum/greytide_worldwide/event = new event_type()
+
+	if(!event.event_setup_start())
+		message_admins("<span class='notice'>[key_name_admin(usr)] tried to use GREYTIDE WORLDWIDE, but it failed in setup. (Type: [event_type_name])</span>", 1)
+		log_admin("[key_name(usr)] tried to use GREYTIDE WORLDWIDE, but it failed in setup. (Type: [event_type_name])")
+		return
+
+	message_admins("<span class='notice'>[key_name_admin(usr)] used GREYTIDE WORLDWIDE! (Type: [event_type_name])</span>", 1)
+	log_admin("[key_name(usr)] used GREYTIDE WORLDWIDE. (Type: [event_type_name])")
+
+	var/list/mobs_to_convert = list()
+	for(var/mob/M in event.eligible_mobs)
+		if(event.check_eligibility(M))
+			mobs_to_convert += M
+
+	for(var/mob/M in mobs_to_convert)
+		event.convert_mob(M)
+
+	event.event_setup_end(mobs_to_convert)
+
+	qdel(event)
+
+
+/datum/greytide_worldwide
+	var/name = ""
+	var/list/eligible_mobs
+
+/datum/greytide_worldwide/New()
+	eligible_mobs = player_list
+
+/datum/greytide_worldwide/proc/event_setup_start()
+	return 1
+
+/datum/greytide_worldwide/proc/check_eligibility(var/mob/M)
+	if(M.stat == DEAD)
+		return 0
+	if(ishuman(M) || issilicon(M))
+		return 1
+
+/datum/greytide_worldwide/proc/convert_mob(var/mob/M)
+	if(issilicon(M))
+		var/mob/living/silicon/S = M
+		var/mob/living/carbon/human/new_human = new /mob/living/carbon/human(S.loc, delay_ready_dna=1)
+		new_human.setGender(pick(MALE, FEMALE))
+		var/datum/preferences/A = new()
+		A.randomize_appearance_for(new_human)
+		new_human.generate_name()
+		new_human.languages |= S.languages
+		if(S.default_language)
+			new_human.default_language = S.default_language
+		if(S.mind)
+			S.mind.transfer_to(new_human)
+		else
+			new_human.key = S.key
+		qdel(S)
+		M = new_human
+	return M
+
+/datum/greytide_worldwide/proc/event_setup_end(var/list/converted_mobs)
+
+
+/datum/greytide_worldwide/greytide
```

Comment:
This is an orphan typepath, there is no real `/datum/greytide_worldwide` and its only subtype is `/datum/greytide_worlwide/greytide`. Remove the `/greytide` subpath, or rename `/datum/greytide_worldwide` to something more generic if you want it as a basis for other events